jormungandr/src/main.rs

```diff
@@ -276,6 +278,8 @@ fn start_services(bootstrapped_node: BootstrappedNode) -> Result<(), start_up::E
         });
     }
 
+    services.spawn_try_future_std("sigint_listener", move |_info| ctrl_c().map_err(|_| ()));
+
```

Comment:
this is unfortunate to do that... :/

Reply:
Actually I can check if can live without that. I was a bit confused by this line from the documentation

> Even if this Signal instance is dropped, subsequent SIGINT deliveries will end up captured by Tokio, and the default platform behavior will NOT be reset.

but Tokio may be able to do some implicit handling.

Reply:
but it doesn't :(